### ISSUE TITLE
Bug when you have a 2 digit mass in your run table

### ISSUE THREAD

**Issue by NicoleRayner** [CIRDLES/Squid#654]
I brought in an old task that included masses 89 to 270.  Squid3 insisted on listing mass 89 at the end of the task list of masses and thus when I wanted to "Manage isotopes" the task labels and data labels did not line up.  On a whim I tried adding a leading 0 to my mass 89 and this indeed took care of it.  Thus it appears the order of the masses in the task list is determined by the FIRST digit in the number, not the absolute number.

![image](https://user-images.githubusercontent.com/40035242/134979903-931a90b7-df83-4c47-9c18-3eac616cb3e7.png)

**Comment by bowring** [comment]
@NicoleRayner - please send along the "old task" for testing, thanks.

**Comment by NicoleRayner** [comment]
here's an "old task" 
[SquidTask_MnzGSC= 12pk Y GSC 8153 std (floating slope)-OVC.BD.zip](https://github.com/CIRDLES/Squid/files/7289335/SquidTask_MnzGSC.12pk.Y.GSC.8153.std.floating.slope.-OVC.BD.zip)
but the same thing happens if I make a task from scratch.  I too initially thought the problem might be the task so when I first encountered this I started from an empty one and added 89, 138 etc and it puts the 89 at the end of the task species list. (only obvious when you ask to use current task).  Just to make sure I wasn't dreaming that  I just repeated this test now:
![image](https://user-images.githubusercontent.com/40035242/136103994-cc2cd46b-396e-42f4-a21d-20a2be547471.png)
